### ISSUE TITLE
Misspelled command name

### ISSUE THREAD

**Issue by ayoubedd** [smravec/nixos-config#3]
At this line.

https://github.com/smravec/nixos-config/blob/f32f40ad80b38222493df36b4e59d79484e44def/nixos/wayland/window-manager.nix#L10C10-L10C10

i guess the command name `dbus-update-activation-enviroment` should be `dbus-update-activation-environment`.

**Comment by smravec** [comment]
Yeah it is misspelled but since it is a custom script that I reference in all the other scripts with this misspelled name it does not break anything. I am too lazy to look up all the references but if you want to change it everywhere and submit a pr I would gladly merge it :)

**Comment by ayoubedd** [comment]
hello again, searching through my notifications lead me back here.

since the `dbus-update-activation-environment` is a command provided by nix there isn't much we can do. 
which made me believe the first line in `/bin/dbus-sway-enviroment` is doing nothing except failing.
